packages/mobile/src/send/SendAmount.tsx

```diff
@@ -260,6 +257,8 @@ function SendAmount(props: Props) {
         addressValidationType,
         isPaymentRequest: true,
       })
+    } else if (recipientVerificationStatus !== RecipientVerificationStatus.VERIFIED) {
+      navigate(Screens.PaymentRequestUnavailable, { transactionData })
```

Comment:
Do we want to fire `CeloAnalytics.track(CustomEventNames.request_continue, continueAnalyticsParams)` as well?